rover_camera/src/glowstick_detection/glowstick_detection_node.hpp

```diff
@@ -0,0 +1,17 @@
+#ifndef GLOWSTICK_DETECTION_NODE_HPP
+#define GLOWSTICK_DETECTION_NODE_HPP
+
+#include "image_capture_glowstick.hpp"
+
+class GlowstickDetectionNode : public rclcpp::Node
+{
+  public:
+    GlowstickDetectionNode();
+    ~GlowstickDetectionNode();
+
+  private:
+    std::unique_ptr<ImageCaptureGlowstick> _camera;
```

Comment:
No need for ptr here. Lets avoid dynamic allocation when we can. Also, variable name _camera is unclear. More like image
```suggestion
    ImageCaptureGlowstick _camera;
```